docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,190 @@
+==============================
+Tutorial: Moving a Data Volume
+==============================
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
+
+.. contents:: 
+   :local:
+   :backlinks: none
+   :depth: 2
+
+You will use Flocker to migrate a Docker container with its data volume from one host to another.
```

Comment:
hosts vs nodes again. Our software uses notes in the `yml` files so change to nodes.